app/src/main/java/com/kylecorry/trail_sense/shared/CustomUiUtils.kt

```diff
@@ -459,4 +459,9 @@ object CustomUiUtils {
         }
     }
 
+    /**
+     * Int extension to convert dp to px based on device pixel density
+     */
+    fun Int.dpToPx(): Int = (this * android.content.res.Resources.getSystem().displayMetrics.density).toInt()
```

Comment:
There's an existing method (from andromeda) for this: Resources.dp(Context, Float)

Can you use that instead of this extension method? 

Here's an example of the usage: https://github.com/kylecorry31/Trail-Sense/blob/4cb25ff92ae624c851410bd186f3415fe53e45a0/app/src/main/java/com/kylecorry/trail_sense/shared/sharing/ActionSheet.kt#L28

Reply:
I'll also revisit my Andromeda dp implementation to see if I can avoid needing the context there - I wasn't aware of that possibility. But that's a larger change, since I would need to update it's usage everywhere and do some regression testing: https://github.com/kylecorry31/andromeda/issues/92

Reply:
Yes, fixed.